src/i18n.tsx

```diff
@@ -1,34 +1,43 @@
 import React from 'react'
 import { i18n } from '@lingui/core'
 import { I18nProvider } from '@lingui/react'
-import { detect, fromUrl } from '@lingui/detect-locale'
-import { ReactNode, useEffect } from 'react'
+import { detect, fromNavigator } from '@lingui/detect-locale'
+import { ReactNode } from 'react'
 
-export const locales = ['en', 'pseudo-en', 'de', 'es-AR', 'es-US', 'it-IT', 'iw', 'ro', 'ru', 'vi', 'zh-CN', 'zh-TW']
-export const defaultLocale = 'en'
+export const supportedLocales = [
+  'en',
+  'pseudo-en',
+  'de',
+  'es-AR',
+  'es-US',
+  'it-IT',
+  'iw',
+  'ro',
+  'ru',
+  'vi',
+  'zh-CN',
+  'zh-TW',
+]
+export const defaultFallback = () => 'en'
```

Comment:
why is this a function now

Reply:
all the examples in lingui use a function so I decided to swtich to a function.